packages/dashboard/app/components/RuntimeFallbackBadge.tsx

```
@@ -0,0 +1,57 @@
+  projectId?: string;
+}
+
+function RuntimeFallbackBadgeComponent({ taskId, isInViewport, projectId }: RuntimeFallbackBadgeProps) {
+  const { addToast } = useToast();
+  const status = useRuntimeFallbackStatus(taskId, isInViewport, projectId);
+
+  useEffect(() => {
+    if (status.shouldToastNow && status.message) {
+      addToast(status.message, "warning");
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps -- addToast identity is stable per ToastProvider instance
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)